lib/ddtrace/contrib/action_view/events/render_template.rb

```diff
@@ -34,6 +34,9 @@ def process(span, _event, _id, payload)
             layout = payload[:layout]
             span.set_tag(Ext::TAG_LAYOUT, layout) if layout
 
+            # Measure service stats
+            Contrib::Analytics.set_measured(span)
```

Comment:
assuming the same question here as `render_partial.rb`

Reply:
Same as other answer.